airbyte-webapp/src/hooks/services/Experiment/experiments.ts

```diff
@@ -1,5 +1,8 @@
 /* eslint sort-keys: "error" */
 /**
+ * Experiments are short-term flags for A/B testing or staged rollouts of features.
+ * They are only available in Airbyte Cloud.
```

Comment:
I find this part of the comment a bit confusing in this file. Since the `useExperiment` hook has a default value which will be used in OSS always, I feel this might confuse new developers to read this as: "don't use experiments in OSS code", which isn't the case.

Reply:
Makes sense, I just removed that part since how the default/fallback experiment value works is not actually relevant to the interface here.